WireAPI/Sources/WireAPI/APIs/ConnectionsAPI/ConnectionsAPIV0.swift

```diff
@@ -0,0 +1,130 @@
+        self.init(httpClient: httpClient, fetchLimit: Constants.maxConnectionsCount)
+    }
+
+    init(httpClient: HTTPClient, fetchLimit: Int) {
+        self.httpClient = httpClient
+        self.fetchLimit = Constants.maxConnectionsCount
+    }
+
+    var apiVersion: APIVersion {
+        .v0
+    }
+
+    var resourcePath: String {
+        "\(pathPrefix)/list-connections/"
+    }
+
+    func fetchConnections() async throws -> PayloadPager<Connection> {
+
+        let pager = PayloadPager<Connection> { start in
+
+            // body Params
+            let params = PaginationRequest(pagingState: start, size: Constants.maxConnectionsCount)
+            let body = try JSONEncoder.defaultEncoder.encode(params)
+
+            let request = HTTPRequest(
+                path: self.resourcePath,
+                method: .post,
+                body: body
+            )
+            let response = try await self.httpClient.executeRequest(request)
+
+            // Parse response
+            let responsePayload = try ResponseParser()
+                .success(code: 200, type: PaginatedConnectionList.self)
+                .failure(code: 400, error: ConnectionsAPIError.invalidBody)
+                .parse(response)
+
+            return PayloadPager<Connection>.Page(
+                element: responsePayload.connections.map { $0.toAPIModel() },
+                hasMore: responsePayload.hasMore,
+                nextStart: responsePayload.pagingState
+            )
+        }
+
+        return pager
+    }
+}
+
+private struct PaginatedConnectionList: Decodable, ToAPIModelConvertible {
+
+     enum CodingKeys: String, CodingKey {
+         case connections
+         case pagingState = "paging_state"
+         case hasMore = "has_more"
+     }
+
+     var nextStartReference: String? {
+         return pagingState
+     }
+
+     let connections: [ConnectionResponseV0]
+     let pagingState: String
+     let hasMore: Bool
+
+    func toAPIModel() -> PaginatedConnectionList {
+        self
+    }
+ }
+
+private struct ConnectionResponseV0: Decodable, ToAPIModelConvertible {
+
+    enum CodingKeys: String, CodingKey {
+        case from
+        case to
+        case qualifiedTo = "qualified_to"
+        case conversationID = "conversation"
+        case qualifiedConversationID = "qualified_conversation"
+        case lastUpdate = "last_update"
+        case status
+    }
+
+    let from: UUID?
+    let to: UUID?
+    let qualifiedTo: QualifiedID?
+    let conversationID: UUID?
+    let qualifiedConversationID: QualifiedID?
+    let lastUpdate: Date
+    let status: ConnectionStatus
+
+    func toAPIModel() -> Connection {
+        Connection(senderId: from,
+                   receiverId: to,
+                   receiverQualifiedId: qualifiedTo,
+                   conversationId: conversationID,
+                   qualifiedConversationId: qualifiedConversationID,
+                   lastUpdate: lastUpdate,
+                   status: status)
```

Comment:
I think we should choose one formatting alignment in the team? Mostly I see this now:

```suggestion
        Connection(
            senderId: from,
            receiverId: to,
            receiverQualifiedId: qualifiedTo,
            conversationId: conversationID,
            qualifiedConversationId: qualifiedConversationID,
            lastUpdate: lastUpdate,
            status: status
        )
```

Reply:
yes let's tackle that generally, not per PR